examples/apps/color_finder.py

```diff
@@ -0,0 +1,659 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+''' Cozmo looks around and drives after colors.
+
+When the program starts, Cozmo will look around for the color blue.
+Tap the cube illuminated blue to switch the color Cozmo will look for.
+Tap the blinking white cube to have the Tkviewer display Cozmo's pixellated camera view.
+
+
+TODO: determine whether to keep pixel_matrix as is, or convert to numpy matrix
+'''
+
+import asyncio
+import functools
+import math
+import numpy
+import sys
+
+import cozmo
+
+from cozmo.util import degrees, radians, distance_mm, speed_mmps
+from cozmo.lights import Color, Light
+try:
+    from PIL import Image, ImageDraw, ImageColor
+except ImportError:
+    sys.exit('Cannot import from PIL: Do `pip3 install --user Pillow` to install')
+
+''' Color utilities:
```

Comment:
Doctrings (with ''') show as the first statement in a module, function, etc. Since these comments are outside a thing, I think instead '#' should be used to lead the lines. Also, let's break it apart so the color_dict description is immediately in front of color_dict, map_color_to_light in front of that method, and then color_distance_sqr can be a docstring inside that method.